gamerpot-back-end/ormconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "type": "mysql",
+  "host": "localhost",
+  "port": 3306,
+  "username": "root",
+  "password": "",
+  "database": "gamerpot-rn",
+  "logging": false,
+  "entities": [
+     "dist/entities/**/*.entity.js"
+  ],
+  "synchronize": true
+}
```

Comment:
I do not recommend that you upload this file to GitHub, instead you can upload an example file for the purpose of hiding values ​​such as username, password, database.